var/spack/repos/builtin/packages/hipblas/package.py

```diff
@@ -15,6 +17,7 @@ class Hipblas(CMakePackage):
     url      = "https://github.com/ROCmSoftwarePlatform/hipBLAS/archive/rocm-4.5.0.tar.gz"
 
     maintainers = ['srekolam', 'arjun-raj-kuppala', 'haampie']
+    libraries = ['libhipblas.so']
```

Comment:
I haven't read the code in detail, but I'd like to reiterate my warning from above again here: libraries don't always end with `.so`, on macOS it's mostly `.dylib`. Is this handled somewhere?

Reply:
The strings placed in the `.libraries` attribute are treated like regular expressions and allow for substring matching. For example if you replaced `libhipblas.so` with `libhipblas`, and Mac OS had `libhipblas.dylib`, this would be able to find it.

I didn't want to require updating the regular expressions for specific packages in this PR: I thought that could occur in future PRs if users on those systems want to autodetect these packages there; in part I did this because I wasn't sure if the PR author had access to a Mac OS installation to confirm the name format for these packages.

Reply:
@nilsvu Yes it would be great to add OX support, but that feels better in a separate PR so that this can merge.